src/lib/y2network/autoinst/hostname_reader.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2network/hostname"
+require "y2network/sysconfig/hostname_reader"
+
+module Y2Network
+  module Autoinst
+    # This class is responsible of importing the AutoYast dns section
```

Comment:
Probably copied from the dns_reader, needs some rewording

Reply:
fixed